types/react-dom-factories/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false
```

Comment:
Same here - the definition seems to have a dt header, so what's the issue here?

Reply:
Same issue as above.